### PR TITLE
Fix the sample script for Unicornafl2

### LINKED ISSUE #1309: Fix Python examples for UnicornAFL2
Python scripts in samples that still need to be fixed:
./speedtest/python/harness.py
./compcov_x64/compcov_test_harness.py
./python_simple/simple_test_harness_alt.py

(plus svg logo for main readme.md)

### PR DESCRIPTION
**Pull request by wtdcode**
This fixes #1309

### CODE REVIEW

File: unicorn_mode/samples/compcov_x64/compcov_test_harness.py [hunk 1]
```diff
@@ -19,7 +19,10 @@
 import signal
 
 from unicornafl import *
-from unicornafl.x86_const import *
+
+monkeypatch()
```
Comment (vanhauser-thc) [inline unicorn_mode/samples/compcov_x64/compcov_test_harness.py]: What is this? :)

Comment (wtdcode) [reply unicorn_mode/samples/compcov_x64/compcov_test_harness.py]: https://github.com/AFLplusplus/unicornafl#migration To make it work both on ucafl1 and ucafl2

Comment (domenukk) [reply unicorn_mode/samples/compcov_x64/compcov_test_harness.py]: We don't need it anymore I think. If you have this file, you'll have unicornafl2?

Comment (wtdcode) [reply unicorn_mode/samples/compcov_x64/compcov_test_harness.py]: Okay fine, let's remove it.

**Comment by domenukk** [comment]
Great, thanks! :)

**Pull request merged**